docs/contribute.md

```diff
@@ -0,0 +1,340 @@
+# Contribute to HashiCorp documentation
+
+> [!NOTE]
+> Click the GitHub UI's **Outline** button, which is next to the **Edit this file** button,
+> to toggle this page's outline.
+
+This repository contains HashiCorp [product documentation and best
+practices](https://developer.hashicorp.com).
+
+The information in this file is generic and applies to documentation
+for products that use versioned docs and create all content in the
+`web-unified-docs` public repository.
+
+Refer to individual product content README files for product-specific
+conventions and processes.
+
+@TODO list out which products this file applies to. Should HCP have its own
+contributing file? What about Terraform (or point to internal wiki pages)?
+
+## Repository structure
+
+Documentation content is written in Markdown. You can find product folders
+in the `content` directory. We publish content from the `main`
+branch.
+
+### Versioned content
+
+Documentation for specific product versions is in folders within the
+`content/<product>` directory.
+
+Note that HCP documentation does not have versions.
+
+### Branch naming conventions
+
+Tech writers create upcoming release branches using these conventions:
+
+- Upcoming major release branch: `<product>/<exact_release_number>`
+- Upcoming minor release branch:  Most products use the
+  `<product>/<exact_release_number>` format, but Vault uses `vault/<YYYYMM>`
+
+Individual contributors should create working branches using one of the following:
+
+- Community contributors:
+  `<github_username>-<product_name>-<github_issue_number>`, such as
+  `aimeeu-nomad-12345`.
+- HashiCorp employees: `<name, initials, or GitHub username>-<ticket_number>`,
+    such as `aimeeu-ce1001`.
+
+## HashiCorp employee contributor guide
+
+> [!IMPORTANT]
+> You must create documentation for embargoed content, including product or
+> conference announcements, in the `web-unified-docs-internal` repository. Contact
+> your team's technical writer for guidance.
+
+### Workflows
+
+@TODO use which presentation??
```

Comment:
List, but get rid of the sentence after the colon, and make each item a link to the workflow.